tests/ui/asm/aarch64/aarch64-sve.rs

```diff
@@ -3,22 +3,10 @@
 //@ needs-asm-support
 
 #![crate_type = "rlib"]
-#![feature(no_core, rustc_attrs, lang_items)]
-#![no_core]
```

Comment:
(I added this test in https://github.com/rust-lang/rust/pull/131332, but no_core was not needed in the first place for this test.)